app/src/main/res/values/strings.xml

```diff
@@ -1206,6 +1206,11 @@ Alternatively, you can leave a note (with a photo)."</string>
     <string name="achievement_lifesaver_solved_X">You solved %d quests that help emergency services and first responders!</string>
     <string name="quest_airConditioning_no_name_title">"Does this place (%s) have air conditioning?"</string>
     <string name="quest_airConditioning_name_type_title">"Does %1$s (%2$s) have air conditioning?"</string>
+    <string name="quest_smoking_no_name_title">"Is smoking allowed anywhere in this place (%s)?"</string>
+    <string name="quest_smoking_name_type_title">"Is smoking allowed anywhere in %1$s (%2$s)?"</string>
+    <string name="quest_smoking_outside">Outside only</string>
+    <string name="quest_smoking_isolated">Prohibited everywhere, except in separate completely isolated smoking room</string>
```

Comment:
```suggestion
    <string name="quest_smoking_isolated">Prohibited everywhere, except in separate isolated smoking room</string>
```

in my experience even decent isolated rooms, passing legislation requirements, have smoke leaving at least when someone enters/leaves it (not aware about systems using airlocks :) )

Reply:
Yeah, I was **_OVEREMPHASIZING_** on purpose there:  People should answer `isolated` **_only if there is foolproof airlock-like system_** (in Croatia, for example, there is no double-door airlock required, but ventilation system is required that sucks air from above with such power to create underpressure in smoking room so even when door opens, the air can move only from outside to inside, not other way around - needless to say very few places choose to actually implement that).

It should be selected only if there is no chance nonsmoker sitting outside of smoking room would be able to be smell the smoke / be bothered about it.  As it happens only very rarely, and introduces possible confusion,  based on those comments I'll likely remove `isolated` option completely. And I'm not sure about `separated` either, see below...